pages/mission.html

```diff
@@ -5,6 +5,7 @@
 banner_file: banner--our-mission-lg.svg
 banner_file_mobile: banner--our-mission-sm.svg
 social_image: social-card--our-mission.png
+social_image_alt_text: "Navy blue graphic block with a yellow star in the upper right corner"
```

Comment:
Suggested text edit:

"Graphic illustration of a five-point, yellow star in the night sky. Rays of blue and gray converge at a horizon line just under the star."